docs/02.API-REFERENCE.md

```diff
@@ -67,7 +67,7 @@ Enum that contains JerryScript **object** value types:
 
 *New in version 2.4*.
 
-*Changed in version [[NEXT_RELEASE]]*: Added `JERRY_OBJECT_TYPE_SCRIPT`, `JERRY_OBJECT_TYPE_MODULE` values.
+*Changed in version [[NEXT_RELEASE]]*: Added `JERRY_OBJECT_TYPE_WEAKREF`, `JERRY_OBJECT_TYPE_SCRIPT`, `JERRY_OBJECT_TYPE_MODULE` values.
```

Comment:
How about `JERRY_OBJECT_TYPE_SHARED_ARRAYBUFFER`? I think it need to be mentioned as well.

Reply:
True.